standard_rules/mfa_disabled.yml

```diff
@@ -8,6 +8,7 @@ LogTypes:
   - Atlassian.Audit
   - GitHub.Audit
   - Zendesk.Audit
+  - Okta.SystemLog
```

Comment:
is the okta data model already included in the standard pack?

Reply:
Yup! Its [here](https://github.com/panther-labs/panther-analysis/blob/master/packs/standard_ruleset.yml#L18)